torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -0,0 +1,276 @@
+#include <ATen/Config.h>
+#include <ATen/code_template.h>
+#include <ATen/core/jit_type.h>
+#include <ATen/native/ConvUtils.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/ir/subgraph_matcher.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/constant_pooling.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/dead_code_elimination.h>
+#include <torch/csrc/jit/passes/fold_conv_bn.h>
+#include <torch/csrc/jit/passes/freeze_module.h>
+#include <torch/csrc/jit/passes/graph_rewrite_helper.h>
+#include <torch/csrc/jit/passes/inliner.h>
+#include <torch/csrc/jit/passes/mkldnn_rewrite.h>
+#include <torch/csrc/jit/passes/remove_dropout.h>
+#include <torch/csrc/jit/runtime/graph_executor_impl.h>
```

Comment:
Do we need all these includes? Please remove those that are not actually used (e.g. `freeze_module.h`).

Reply:
Thanks for pointing that out. I've removed headers that are not actually needed in this file (https://github.com/pytorch/pytorch/pull/77157/commits/ebeb5792167eecbee051955080b4ab24c64a45a5)